rss/factory.py

```diff
@@ -0,0 +1,39 @@
+"""Application factory for rss app."""
+
+
+from flask import Flask
+from celery import Celery
+
+from arxiv.base import logging
+
+from rss import celeryconfig
+from rss import routes
+#from rss.services import baz
+#from rss.encode import ISO8601JSONEncoder
+#from rss.middleware import auth
+from arxiv.base.middleware import wrap
+from arxiv.base import Base
+
+
+celery_app = Celery(__name__, results=celeryconfig.result_backend,
+                    broker=celeryconfig.broker_url)
+
+
+def create_web_app() -> Flask:
+    """Initialize and configure the rss application."""
+    app = Flask('rss')
+    app.config.from_pyfile('config.py')
+    #app.json_encoder = ISO8601JSONEncoder
+
+    #baz.init_app(app)
+
+    Base(app)    # Gives us access to the base UI templates and resources.
+    app.register_blueprint(routes.blueprint)
+
+    celery_app.config_from_object(celeryconfig)
+    celery_app.autodiscover_tasks(['rss'], related_name='tasks', force=True)
+    celery_app.conf.task_default_queue = 'rss-worker'
```

Comment:
I agree with @bbarker , not sure whether you'll need the Celery bits, since you're just reading from ElasticSearch (unless I'm forgetting something)